### DISCUSSION TITLE
how to create a simple chart with livecharts 2

### DISCUSSION THREAD

**Discussion by humbertomoli99** [Live-Charts/LiveCharts2#203] [Q&A] [answered]
Hello, I am very interested in the package but there is no good documentation and the code examples are difficult for me to understand.
especially because my app is for uwp and I don't know how to use the packages.

**Comment by CodeWithMichal** [comment, marked as answer]
Hi, 
Did you check documentation in this folder? https://github.com/beto-rodriguez/LiveCharts2/tree/master/docs

What exactly is missing in documentation?
Also, there are samples for UWP there:
https://github.com/beto-rodriguez/LiveCharts2/tree/master/samples/UWPSample
With associated view models there:
https://github.com/beto-rodriguez/LiveCharts2/tree/master/samples/ViewModelsSamples

If you are not familiar with MVVM pattern (view models) then you need to read about it first to fully understand samples

**Comment by Mielesgames** [comment]
You should check out this website, I have no idea if its official, but it has a bunch of tutorials
https://lvcharts.com/docs/maui/2.0.0-beta.330/Overview.Installation
